files/start.sh

```diff
@@ -17,7 +17,8 @@ com.cloudera.cmf.db.password=${DB_PASSWORD}
 EOF
 
 ## Default env
-source /etc/default/cloudera-scm-server
+export CMF_JDBC_DRIVER_JAR=${CMF_JDBC_DRIVER_JAR:-/usr/share/java/mysql-connector-java.jar:/usr/share/java/oracle-connector-java.jar}
+export CMF_JAVA_OPTS=${CMF_JAVA_OPTS:--Xmx2G -XX:MaxPermSize=256m -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp -Dcmf.root.logger=INFO,LOGFILE,CONSOLE}
```

Comment:
The significant changes here are that:
1) We only apply default config if these env vars aren't already defined
2) root logger includes *both* logfile and console ( default is logfile only )